guestbook/php-redis/Dockerfile

```diff
@@ -12,7 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM php:5-apache
+FROM BASEIMAGE
```

Comment:
I'm thinking just to eliminate confusion, maybe we should maintain 2 copies: Dockerfile and Dockerfile.multiarch?